languages/ruby/reference/exercise-concepts/resistor-color-duo.md

```diff
@@ -0,0 +1,51 @@
+# Concepts of Resistor Color Duo
+
+## General
+
+ [Example solution](https://github.com/exercism/ruby/blob/master/exercises/resistor-color-duo/.meta/solutions/resistor_color_duo.rb)
+
+- Classes:
+  - Defining classes
+- Methods:
+  - Defining methods
+  - Arguments
+  - Return values
+- Strings:
+  - Basics like difference between `1` and `"1"`
+- Class methods:
+  - How to define a method so it can be called directly on a class
```

Comment:
```suggestion
  - How to define a class method
```

Where do we talk about defining methods?  Should it be here or should it be covered in a more basic place.  I ask because this should be discovered well before this exercise.